common/display/src/test/scala/uk/ac/wellcome/display/models/DisplayCollectionTest.scala

```diff
@@ -184,4 +185,38 @@ class DisplayCollectionTest extends FunSpec with Matchers with WorksGenerators {
         )
       )
   }
+
+  it("doesn't set expanded to paths sharing the same prefix as expanded") {
```

Comment:
I’m confused – how does this test prove the issue is fixed?

Is it because `a/b` isn't expanded (`children` is `None`)? Wouldn’t that happen with the old behaviour?

If you were looking at `a/b`, you'd previously see `a/b0` expanded as well, but I'm not sure the inverse is true.

Reply:
> Is it because a/b isn't expanded (children is None)? Wouldn’t that happen with the old behaviour?

Nope the old behaviour children was be `Some(List())` rather than `None`. Does that make sense?

Reply:
Ah, yeah, I think I'd misunderstood the original issue. Got it!